apps/web/src/components/comment/comment.tsx

```diff
@@ -7,6 +7,7 @@ import { deleteEventCommentById } from '@/api/event'
 import CommentOption from '@/components/comment/commentOption'
 import CommentUpdateField from '@/components/comment/commentUpdateField'
 import { UnlikeIcon } from '@/components/icon'
+import LoadingOverlay from '../loadingOverlay'
```

Comment:
```suggestion
import LoadingOverlay from '@/component/loadingOverlay'
```
위와 같이 통일시켜 주면 좋을 것 같아요